ios/DomainLayer/SharedDomain/Sources/SharedDomain/StoreKit/UseCases/RequestFeedbackUseCase.swift

```diff
@@ -19,12 +19,6 @@ public struct RequestFeedbackUseCaseImpl: RequestFeedbackUseCase {
     }
     
     public func execute() throws {
-        if Environment.flavor == .debug {
-            guard CommandLine.arguments.contains("-ShouldShowFeedback") else { return }
-            
-            try storeKitRepository.requestFeedback()
-        } else {
-            try storeKitRepository.requestFeedback()
-        }
```

Comment:
Why this change? Please discuss with @DavidKadlcek, as he introduced this code in https://github.com/MateeDevs/devstack-native-app/pull/99#issue-2076903007

Reply:
Ahhhh, it was crashing at this bit, David already fixed it in his branch, I think, this was just to help me get to my tests and I forgot to remove this change, sorry

Reply:
I added @DavidKadlcek 's solution as a new commit, now all tests work 🙂